mock-data/CosmosDbSeeder/items/index_data/global_roles.json

```diff
@@ -36,6 +36,19 @@
       "description": "A user can view and manage user role assignments.",
       "permissions": ["users:manage", "users:view"]
     },
+    {
+      "id": "canned:auditer",
+      "name": "Auditer",
+      "description": "A user who can see everything (including admin-level data) but take no actions. This is a read-only role.",
+      "permissions": [
+        "site:get-config",
+        "site:get-meta-data",
+        "availability:get-setup",
+        "availability:query",
+        "booking:query",
+        "users:view"
+      ]
+    },
```

Comment:
As mentioned in the description, I anticipate a bit of discussion around this. Happy to make it a system role or whatever, I just need something with `users:view` but not `users:manage`.

Reply:
I think we have the flexibility to build up whatever roles we need. IMO I don't think for testing purposes we don't need to stick to the canned roles as defined in confluence. For the integration tests we created a role with the permissions need for running those so I don't see any issue with doing the same for the e2e testing pursposes.